src/licenses/fetch.rs

```diff
@@ -84,9 +84,10 @@ impl GitHostFlavor {
 #[serde(deny_unknown_fields)]
 pub struct GitInfo {
     pub sha1: String,
+    pub dirty: bool,
```

Comment:
This change led to reports like

> [WARN] failed to validate all files specified in clarification for crate const-str 0.6.2: unable to retrieve 'LICENSE' for crate 'const-str 0.6.2' from remote git host: failed to deserialize '/Users/jrose/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/const-str-0.6.2/.cargo_vcs_info.json': missing field `dirty` at line 4 column 3

when attempting to update to cargo-about 0.8.0. Perhaps it should be marked `#[serde(default)]`?